tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -667,6 +657,11 @@ class MklMatMulPrimitive : public MklPrimitive {
     if (add_data != nullptr) context_.add_mem->set_data_handle(DummyData);
   }
 
+  std::shared_ptr<dnnl::matmul::primitive_desc>
+  GetPrimitiveDesc() const {
+    return context_.prim_desc;
+  }
```

Comment:
@TensorFlow-MKL This change also affects the x86 build, although I don't think it can do any harm.